universal-ctags.rb

```diff
@@ -1,17 +1,24 @@
 class UniversalCtags < Formula
   homepage 'https://github.com/universal-ctags/ctags'
   head 'https://github.com/universal-ctags/ctags.git'
+
+  option "without-iconv", "Enable multibyte character encoding"
```

Comment:
Ops, that's a mistake. Because initially I made iconv disabled by default and added the option to disable it. Then I changed my mind and decided it was better to enable it but I forgot to change the help text to tell that passing that option would disable multibyte support.